pkg/genericapiserver/genericapiserver.go

```diff
@@ -208,8 +208,12 @@ type Config struct {
 	// allow downstream consumers to disable the core controller loops
 	EnableLogsSupport bool
 	EnableUISupport   bool
```

Comment:
Renaming the option to EnableSwaggerUI would cause Openshift and other apiserver clients to not compile, so let's not do that. We should document what the option does, though.